zeapp/desktop/src/main/kotlin/de/berlindroid/zekompanion/desktop/ui/NameBadgeEditor.kt

```diff
@@ -31,6 +31,8 @@ import androidx.compose.ui.unit.IntSize
 import androidx.compose.ui.unit.TextUnit
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import de.berlindroid.zekompanion.BADGE_HEIGHT
+import de.berlindroid.zekompanion.BADGE_WIDTH
```

Comment:
I'm not sure if that change is needed though. Shouldn't it be included already?

Reply:
Dunno, refused to build without